net/vnstat/src/opnsense/mvc/app/controllers/OPNsense/Vnstat/Api/ServiceController.php

```diff
@@ -99,4 +92,31 @@ public function resetdbAction()
         $response = $backend->configdRun("vnstat resetdb");
         return array("response" => $response);
     }
+
+    /**
+     * Get stats of the given type
+     * @param string $type Which type of statistics to return (hourly, daily, weekly, or monthly)
+     * @return array
+     */
+    private function getStats(string $type) {
+        $config = Config::getInstance()->toArray();
+        $backend = new Backend();
+
+        if (isset($config['OPNsense']['vnstat']['general']['separate_stats']) && !$config['OPNsense']['vnstat']['general']['separate_stats']) {
```

Comment:
Something like `$config?->OPNsense?->vnstat?->etc` should work for what you're trying to do here